src/style.css

```
@@ -1,3 +1,109 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --hover-color: rgba(0, 172, 220, 1);
+}
+
+header {
```

Use only class selector, fix it everywhere
```suggestion
.header {
```